base/src/java/util/HashSet.d

```diff
@@ -10,78 +10,187 @@ version(Tango){
 } else { // Phobos
 }
 
+/**
+    This class implements the `Set` interface, backed by a hash table.
+
+    It makes no guarantees as to the iteration order of the set; in
+    particular, it does not guarantee that the order will remain
+    constant over time.  This class permits the `null` element.
+*/
 class HashSet : Set {
     version(Tango){
         alias tango.util.container.HashSet.HashSet!(Object) SetType;
         private SetType set;
     } else { // Phobos
+        alias SetType = Object[Object];

```

Comment:
Will do. Originally chose `Object` since I think that's what Java used/uses.

Reply:
I see. Since the user never will see the value, there's no need to match Java in this case. It's also unnecessary to heap allocate a dummy value every time.